sopel_waifu/waifu.json

```
@@ -1468,47 +1470,106 @@
         "Kazusa Touma"
     ],
     "Working!!": [
-        "Popura \"Poplar\" Taneshima",
-        "Aoi Yamada",
-        "Kyouko Shirafuji",
-        "Mahiru Inami",
-        "Maya Matsumoto",
-        "Yachiyo Todoroki",
-        "Izumi Takanashi",
-        "Kazue Takanashi",
-        "Kozue Takanashi",
-        "Nazuna Takanashi"
+        "Taneshima Popura",
+        "Inami Mahiru",
+        "Shirafuji Kyouko",
+        "Todoroki Yachiyo",
+        "Yamada Aoi",
+        "Takanashi Izumi",
+        "Takanashi Kazue",
+        "Takanashi Kozue",
+        "Takanashi Nazuna",
+        "Matsumoto Maya",
+        "Mashiba Mitsuki",
+        "Otoo Haruna",
+        "Kitahara Ritsuko",
+        "Suzutani Momoka"
     ],
     "Yahari Ore no Seishun Love Comedy wa Machigatteiru": [
-        "Yui Yuigahama",
-        "Yukino Yukinoshita",
-        "Komachi Hikigaya"
+        "Yukinoshita Yukino",
+        "Yuigahama Yui",
+        "Isshiki Iroha",
+        "Hikigaya Komachi",
+        "Hiratsuka Shizuka",
+        "Kawasaki Saki",
+        "Yukinoshita Haruno",
+        "Miura Yumiko",
+        "Ebina Hina",
+        "Tsurumi Rumi",
+        "Shiromeguri Meguri",
+        "Orimoto Kaori",
+        "Nakamachi Chika",
+        "Kawasaki Keika",
+        "Yuka",
+        "Hitomi",
+        "Mori",
+        "Yokko",
+        "Sagami Minami"
     ],
     "Yokohama Kaidashi Kikou": [
-        "Hatsuseno Alpha"
+        "Hatsuseno Alpha",
+        "Takatsu Kokone"
     ],
     "Yosuga no Sora": [
-        "Kasugano Sora"
+        "Amatsume Akira",
+        "Yorihime Nao",
+        "Migiwa Kazuha",
+        "Kasugano Sora",
+        "Nogisaka Motoka",
+        "Kuranaga Kozue",
+        "Ifukube Yahiro"
     ],
     "Yuru Yuri": [
         "Toshinou Kyouko",
-        "Akaza \"Who?\" Akari",
+        "Akaza Akari",
```

Did you not watch this show? It's a very relevant joke, probably the best nickname in the whole original list.

```suggestion
        "Akaza \"Who?\" Akari",
```